lib-ltc/include/h3/ltcoutputs.h

```diff
@@ -42,6 +43,21 @@ class LtcOutputs {
 		m_tTimeCodeTypePrevious = ltc::type::INVALID;
 	}
 
+
+	TLtcTimeCode* GetNowTimeCode() {
+		return &m_pNowLtcTimeCode;
+	}
+
+	TLtcTimeCode* GetStopTimeCode() {
+		return &m_pStopLtcTimeCode;
+	}
+
+	void SetStopTimeCode(TLtcTimeCode* tc) {
+		if (tc) {
```

Comment:
tc is not a bool type

Reply:
Just a habit of taking in ptr, it should not get call with a nullptr if used properly.

Real question here,  faster/prefer to just assign the 5 uint8_t values in TLtcTimeCode rather than use memcpy?

```
if (tc != nullptr) {
  memcpy(&m_pStopLtcTimeCode, tc, sizeof(TLtcTimeCode));  // better to just assign 5 bytes, or compiler optimize?
} 
```